### ISSUE TITLE
Update the rosaria installation procedure

### ISSUE THREAD

**Issue by dmiklic** [amor-ros-pkg/rosaria#26]
As the libaria-dev package has become available through official [debian](https://packages.debian.org/search?keywords=libaria-dev) and [ubuntu](http://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=libaria-dev&searchon=names) repositories, the current installation procedure as defined in [libaria.rdmanifest](../blob/master/libaria.rdmanifest) should be updated and changes should be propagated to [rosdep](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml).

**Comment by NikolausDemmel** [comment]
Any updates here?

**Comment by dmiklic** [comment]
Hi Nikolaus, just I haven't found the time for this so far, but I'm updating the software on our Pioneers right now, so I'm planning to handle this issue within the next couple of weeks.

**Comment by dmiklic** [comment]
I have updated [rosdep] (ros/rosdistro@84c72a2217d08264d295e5a86d48a72d11f4e79c) install rules for libaria to use apt packages where they are available. I still need to repackage the latest libaria release and master the repackaging process along the way.

**Comment by dmiklic** [comment]
Hi @reed-adept sorry for letting this sit for so long. I'll repackage the latest Aria version for Debian, please point me to the correct download location.

**Comment by dmiklic** [comment]
Is 2.9.1. the correct version, available from [here](http://robots.mobilerobots.com/wiki/ARIA)?

**Comment by reed-adept** [comment]
Yes 2.9.1 is the most recent version.  Unless a bug is found there won't be another release for at least a few months.   In previous versions there was a ".tgz" package  that contained both source code and compiled binaries.  I've replaced this with a source compressed tar archive (.tar.gz) package (and Zip) that does not contain binaries (though it does contain some other  generated files, not sure if that will be changed in the future.)   There are now 32-bit and 64-bit packages for Ubuntu 16 as well, but these are still not Debian package compliant.  (Thinking about doing that someday but it is low priority.)

**Comment by reed-adept** [comment]
Change log is http://robots.mobilerobots.com/ARIA/download/current/Changes.txt

**Comment by reed-adept** [comment]
Has anyone repackaged ARIA 2.9.1?   (Or put code in libaria.rdmanifest that installs the source .tar.gz package?)

**Comment by dmiklic** [comment]
Sorry Reed, I still haven't gotten around to repackaging ARIA 2.9.1 :/ It's on my todo list until the end of this month. If I don't manage to do it by then, let's discuss reverting to the old approach, ok?

**Comment by dmiklic** [comment]
@reed-adept I'm working on this (actively now :) I  still need some pointers from @jksrecko but I think I'll be able to get 2.9.1 into Debian by the end of the week. You can follow the progress at https://github.com/amor-ros-pkg/libaria

**Comment by reed-adept** [comment]
OK great, glad to hear it.  Did I mention to you  that there is now a tgz of ARIA with just source code (not precompiled), though it is still not in standard installation layout yet and does have a few generated files (namely API docs) that you might not want?   I’ll review tutorials and documentation when the new package is ready if you want.

**Comment by dmiklic** [comment]
Yes, the source-only version is the one I'm using right now for the release. I have removed the remaining generated files manually. After the release is complete, I'll get back to you with the list of manual steps that were still necessary between your source release and the Debian release, and we can discuss how we could make this gap even smaller.

**Comment by dmiklic** [comment]
Srecko is quite busy at the moment, so it will take another week or two before we finalize this, but it will happen this month :)
